frontend/src/components/WaiverComponent/Waiver.jsx

```diff
@@ -6,25 +6,25 @@ import releaseWaiver from "./WAIVER FORM 2021.docx.pdf";
 pdfjs.GlobalWorkerOptions.workerSrc = `https://cdnjs.cloudflare.com/ajax/libs/pdf.js/${pdfjs.version}/pdf.worker.js`;
 
 const InnerWaiver = styled.div`
-  background: #c4c4c4;
+  padding: 10px;
   border: 10px;
-  width: 47.81%;
-  height: 100vh;
   overflow-x: scroll;
+  overflow-y: scroll;
+  transform: scale(0.85);
 `;
 
-const StyledPage = styled(Page)`
-  & .react-pdf__Page__canvas {
-    width: 94.27vh !important;
-    height: 122vh !important;
-  }
-`;
+// const StyledPage = styled(Page)`
+//   & .react-pdf__Page__canvas {
+//     width: 94.27vh !important;
+//     height: 122vh !important;
+//   }
+// `;
```

Comment:
Feel free to delete instead of commenting out

Reply:
Forgot to do that earlier, but I just went through and removed the comments from the WaiverPage and Waiver files